doc/classes/Curve3D.xml

```diff
@@ -132,6 +132,15 @@
 				If the curve has no up vectors, the function sends an error to the console, and returns [code](0, 1, 0)[/code].
 			</description>
 		</method>
+		<method name="sample_baked_with_rotation" qualifiers="const">
+			<return type="Transform3D" />
+			<param index="0" name="offset" type="float" />
+			<param index="1" name="cubic" type="bool" default="false" />
+			<param index="2" name="apply_tilt" type="bool" default="false" />
+			<description>
+				Similar with [code]interpolate_baked()[/code]. The the return value is [code]Transform3D[/code], with [code]origin[/code] as point position, [code]basis.x[/code] as sideway vector, [code]basis.y[/code] as up vector, [code]basis.z[/code] as forward vector. However when [code]get_backed_length() == 0[/code] the return value is a blank, as there is no reasonable way to caculate the rotation.
```

Comment:
This paragraph is outdated and can be updated.

Reply:
I made some change. But I am not sure if it is clear enough. If you have suggestion feel free modify directly.